### PR TITLE
feat: update build scripts

### PR DESCRIPTION
**Pull request by boojack**
The vite config `build.emptyOutDir` has an unwanted behavir in building, so that I have to write a script to control the empty out dir.

### CODE REVIEW

**Comment by boojack** [comment]
PTAL

File: vite.config.ts [hunk 1]
```diff
@@ -1,15 +1,16 @@
 import { defineConfig, UserConfigExport } from "vite";
 import vue from "@vitejs/plugin-vue";
-import generateSitemap from "./scripts/generateSitemap";
 import copyExtensionFiles from "./scripts/copyExtensionFiles";
+import emptyOutDir from "./scripts/emptyOutDir";
+import generateSitemap from "./scripts/generateSitemap";
 
 // https://vitejs.dev/config/
 export default defineConfig(({ command, mode }) => {
+  emptyOutDir();
```
Comment (tianzhou) [inline vite.config.ts:9]: Add a comment here to explain the reason

**Review by tianzhou — APPROVED** [review]
LGTM. One minor comment

File: vite.config.ts (hunk 1 above)
Comment (boojack) [reply vite.config.ts:11]: Already done, PTAL again.

File: vite.config.ts [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
 import vue from "@vitejs/plugin-vue";
-import generateSitemap from "./scripts/generateSitemap";
 import copyExtensionFiles from "./scripts/copyExtensionFiles";
+import emptyOutDir from "./scripts/emptyOutDir";
+import generateSitemap from "./scripts/generateSitemap";
 
 // https://vitejs.dev/config/
 export default defineConfig(({ command, mode }) => {
+  // Control the building behavior and make its first step is to empty the output folder.
+  // To make sure the other script can works right.
```
Comment (tianzhou) [inline vite.config.ts:10]:
```suggestion
  // Control the building behavior and make its first step to empty the output folder.
  // This is to make sure the other script can work right.
```

**Pull request merged**